src/scripts/main.js

```diff
@@ -1,3 +1,332 @@
+    rows.forEach((r) => r.classList.remove('active'));
+
+    row.classList.add('active');
+  });
+});
+
+// const form = document.createElement('form');
+
+// form.classList.add('new-employee-form');
+
+// const nameLabel = document.createElement('label');
+
+// nameLabel.innerHTML =
+//   'Name: <input name="name" type="text" data-qa="name" required>';
+// form.appendChild(nameLabel);
+
+// const positionLabel = document.createElement('label');
+
+// positionLabel.innerHTML =
+//   'Position: <input name="position" type="text" data-qa="position">';
+// form.appendChild(positionLabel);
+
+// const ageLabel = document.createElement('label');
+
+// ageLabel.innerHTML =
+//   'Age: <input name="age" type="number" data-qa="age" required>';
+// form.appendChild(ageLabel);
+
+// const salaryLabel = document.createElement('label');
+
+// salaryLabel.innerHTML =
+//   'Salary: <input name="salary" type="number" data-qa="salary" required>';
+// form.appendChild(salaryLabel);
+
+// const officeSelectLabel = document.createElement('label');
+
+// officeSelectLabel.innerHTML = 'Office: ';
+
+// const officeSelect = document.createElement('select');
+
+// officeSelect.name = 'office';
+// officeSelect.setAttribute('data-qa', 'office');
+
+// const cities = [
+//   'Tokyo',
+//   'Singapore',
+//   'London',
+//   'New York',
+//   'Edinburgh',
+//   'San Francisco',
+// ];
+
+// cities.forEach((city) => {
+//   const option = document.createElement('option');
+
+//   option.value = city;
+//   option.textContent = city;
+//   officeSelect.appendChild(option);
+// });
+
+// officeSelectLabel.appendChild(officeSelect);
+// form.appendChild(officeSelectLabel);
+
+// const button = document.createElement('button');
+
+// button.type = 'submit';
+// button.textContent = 'Save to table';
+// form.appendChild(button);
+
+// document.body.appendChild(form);
+
+const formHTML = `
+  <form class="new-employee-form">
+    <label>
+      Name: <input name="name" type="text" data-qa="name" required>
+    </label>
+    <label>
+      Position: <input name="position" type="text" data-qa="position">
+    </label>
+    <label>
+      Age: <input name="age" type="number" data-qa="age" required>
+    </label>
+    <label>
+      Salary: <input name="salary" type="number" data-qa="salary" required>
+    </label>
+    <label>
+      Office:
+      <select name="office" data-qa="office">
+        <option value="Tokyo">Tokyo</option>
+        <option value="Singapore">Singapore</option>
+        <option value="London">London</option>
+        <option value="New York">New York</option>
+        <option value="Edinburgh">Edinburgh</option>
+        <option value="San Francisco">San Francisco</option>
+      </select>
+    </label>
+    <button type="submit">Save to table</button>
+  </form>
+`;
+
+document.body.insertAdjacentHTML('beforeend', formHTML);
+
+const form = document.querySelector('.new-employee-form');
+
+form.addEventListener('submit', (e) => {
```

Comment:
The function `validateForm` is doing more than validating the form. It's also adding an employee to the table and showing a notification. Consider renaming this function to reflect this, or better yet, separate the concerns of form validation, adding an employee, and showing a notification into different functions.